### ISSUE TITLE
Add screen reader utility class

### ISSUE THREAD

**Issue by lijunle** [OfficeDev/Office-UI-Fabric#439]
Something like `sr-only` class in bootstrap.

```css
.sr-only {
    position: absolute;
    width: 1px;
    height: 1px;
    padding: 0;
    margin: -1px;
    overflow: hidden;
    clip: rect(0,0,0,0);
    border: 0;
}
```

**Comment by mikewheaton** [comment]
Foundation also has [visibility classes for screen readers](http://foundation.zurb.com/sites/docs/v/5.5.3/components/visibility.html#accessibility) and this is something we should absolutely include, along with more guidance for accessibility.

**Comment by mikewheaton** [comment]
Fixed by #534. 😄
